DocumentedExamples/Regridding.ipynb

```diff
@@ -0,0 +1,2541 @@
+       "       ...,\n",
+       "       [ 87.5,  87.5,  87.5, ...,  87.5,  87.5,  87.5],\n",
+       "       [ 88.5,  88.5,  88.5, ...,  88.5,  88.5,  88.5],\n",
+       "       [ 89.5,  89.5,  89.5, ...,  89.5,  89.5,  89.5]])</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-b94f8343-9d33-416f-bf9d-8f38cec1e85c' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-b94f8343-9d33-416f-bf9d-8f38cec1e85c' class='xr-section-summary'  title='Expand/collapse section'>Data variables: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-88d139bd-1bd4-4f6c-b634-eb003d617102' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-88d139bd-1bd4-4f6c-b634-eb003d617102' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:    (x: 360, y: 180)\n",
+       "Coordinates:\n",
+       "    longitude  (y, x) float64 -279.5 -278.5 -277.5 -276.5 ... 77.5 78.5 79.5\n",
+       "    latitude   (y, x) float64 -89.5 -89.5 -89.5 -89.5 ... 89.5 89.5 89.5 89.5\n",
+       "Dimensions without coordinates: x, y\n",
+       "Data variables:\n",
+       "    *empty*"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ds_out = xe.util.grid_global(1, 1)\n",
+    "ds_out = ds_out.drop({'lon_b', 'lat_b'})\n",
+    "ds_out = ds_out.assign_coords({'lon': ds_out.lon - 100.})\n",
+    "ds_out = ds_out.rename({'lon': 'longitude', 'lat': 'latitude'})\n",
+    "ds_out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we construct the three regridders for the three different resolutions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "xESMF Regridder \n",
+       "Regridding algorithm:       bilinear \n",
+       "Weight filename:            bilinear_tracer_weights_in1degACCESSOM2_out1deg.nc \n",
+       "Reuse pre-computed weights? False \n",
+       "Input grid shape:           (300, 360) \n",
+       "Output grid shape:          (180, 360) \n",
+       "Periodic in longitude?      True"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "regridder_1degACCESSOM2_1deg = xe.Regridder(ssh_1.drop({'x', 'y'}), ds_out, 'bilinear', periodic=True, filename=\"bilinear_tracer_weights_in1degACCESSOM2_out1deg.nc\")# , reuse_weights=True)\n",
```

Comment:
(and other regridder lines) remove commented-out `reuse_weights` (or uncomment, I'm not sure how long creating the weights is actually taking for these)